### DISCUSSION TITLE
Stoper

### DISCUSSION THREAD

**Discussion by marekdabrowski** [HanSolo/Medusa#218] [General]
Hello 

I try create application to visualize time from some event (time when some file appear in catalog) and fire alarm (e.g. red background) when time from even to now is greater (1min, 1hour, depends ... )
For example file appears at 2022.06.20 10:00 and gaguge/clock have to show that file was appear some minutes ago and grows, but if e.g. counter will be greater than 2 hour background will change to visualize lack of new files. 

I try do it but my java knowlage is to smal.

Regards
Marek

**Comment by HanSolo** [comment]
Hi Marek, 
Well I think you should learn some Java first before you try this kind of stuff. I have not the time to write your code, so the best thing to do would be to take a look at the [Demo.java](https://github.com/HanSolo/medusa/blob/master/src/main/java/eu/hansolo/medusa/Demo.java) file which contains the code to run the demo. As soon as you understand that code you will be able to create the stuff you have asked for. In principle this is not really hard to do but you need at least some knowledge about Java and JavaFX. You might want to start with taking the Demo class and strip it down to just one gauge and with that you can then start playing around. Good luck and enjoy learning something new :)
Cheers, 
Gerrit

**Comment by marekdabrowski** [comment]
Ok. Thank You.

pon., 20 cze 2022, 14:19 użytkownik Gerrit Grunwald <
***@***.***> napisał:
